### PR TITLE
Collecting release tap files

### LINKED ISSUE #5242: Simplify the process of final AQAvit results archives for release
Simplify the process of final AQAvit results archive can alleviate manual and cumbersome effort. Current process is mostly done manually.

1. Download the results per platform by link provided by jenkins (example https://ci.adoptium.net/job/build-scripts/job/release-openjdk17-pipeline/62/artifact/target/aix/ppc64/temurin/AQAvitTaps/*zip*/AQAvitTaps.zip)
2. Download the rerun jobs' and grinder jobs' tap files and upload to release triage issue
3. Rerun failed jobs ( aborted jobs ) and download the tap files and upload to release triage issue
4. Unzip the zip file and download tap files attached to release triage issue
5. Zip all tapfiles and upload as a jenkins archive.

This can be simplified by

1. Upstream pipeline job update to provide AQAvit results per release (include taps from rerun jobs https://github.com/adoptium/aqa-tests/issues/5015) https://github.com/adoptium/ci-jenkins-pipelines/pull/1027
2. Separate job to download one zip file from upstream pipeline, grab all triage generated tap files attached to the release triage issue, archived artifacts which can be accessed by url (https://github.com/adoptium/github-release-scripts/issues/155) https://github.com/adoptium/aqa-tests/pull/5404 

The final zip file can also be verified https://github.com/adoptium/aqa-tests/issues/3713 before publish.

### PR DESCRIPTION
**Pull request by sophia-guo**
Close #5242 

Taps files attached to release triage issue
Taps Tar files of release pipeline line jobs

Archived - can be consumed by verification jobs as jenkins 's archive artifacts ( url )

### CODE REVIEW

**Comment by sophia-guo** [comment]
Example:https://ci.adoptium.net/view/work-in-progress/job/Sophia_pipeline/614/parameters/

File: buildenv/jenkins/tapVerification/TapCollection.groovy [hunk 1]
```diff
@@ -0,0 +1,88 @@
+pipeline {
+    agent { label 'ci.role.test&&hw.arch.x86&&sw.os.linux' }
+    parameters {
+        string(name: 'RELEASE_TRIAGE_ISSUE_NUMBER', defaultValue: '', description: 'Triage issue number')
+        string(name: 'RELEASE_TRIAGE_REPO', defaultValue: 'adoptium/aqa-tests', description: 'Triage repo')
```
Comment (smlambert) [inline buildenv/jenkins/tapVerification/TapCollection.groovy:5]: Do you mind to put the TRIAGE_REPO first, and the TRIAGE_ISSUE_NUMBER after it?  Matches the ordering we use in other pipelines and in the case of jobName followed by jobNumber

**Review by smlambert — APPROVED** [review]
Other than the order of the parameters comment I added, this LGTM.

File: buildenv/jenkins/tapVerification/TapCollection.groovy (hunk 1 above)
Comment (sophia-guo) [reply buildenv/jenkins/tapVerification/TapCollection.groovy]: Updated

File: buildenv/jenkins/tapVerification/TapCollection.groovy [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                    }
+                    //Download Taps from upsteam 
+                    def builds = "${Release_PipelineJob_Numbers}".split(',')
+                    def tapTars = "AQAvitTapFiles.tar.gz"
+                    builds.each { build ->
+                        echo "build is ${build}"
+                        copyArtifacts (
+                            filter: "AQAvitTaps/${tapTars}", 
+                            fingerprintArtifacts: true, 
+                            projectName: "build-scripts/openjdk21-pipeline",  //"${Release_PipelineJob_Name}"
```
Comment (andrew-m-leonard) [inline buildenv/jenkins/tapVerification/TapCollection.groovy:58]: "${Release_PipelineJob_Name}" maybe ?

Comment (sophia-guo) [reply buildenv/jenkins/tapVerification/TapCollection.groovy:58]: Yes, thanks!

**Review by andrew-m-leonard — APPROVED** [review]
lgtm

**Comment by sophia-guo** [comment]
Job created as https://ci.adoptium.net/view/Test_grinder/job/TAP_Collection

**Pull request merged**